android/PhysicalWeb/app/src/main/res/layout/list_item_nearby_beacon.xml

```diff
@@ -19,24 +19,35 @@
 
         <LinearLayout
                 android:layout_width="0dp"
-                android:layout_height="wrap_content"
+                android:layout_height="76dp"
                 android:orientation="vertical"
                 android:layout_weight="1"
-                android:gravity="start">
+                android:gravity="start"
+            android:weightSum="1">
             <ImageView
                     android:id="@+id/icon"
                     android:layout_width="24dp"
                     android:layout_height="24dp"
                     android:layout_marginTop="6dp"
                     android:contentDescription="@string/nearby_beacon_favicon_content_description">
             </ImageView>
+
+            <ImageView
+                android:id="@+id/star"
+                android:layout_width="30dp"
+                android:layout_height="30dp"
+                android:layout_marginTop="6dp"
+                android:contentDescription="@string/nearby_beacon_favicon_content_description"
+                android:clickable="true"
+                android:layout_weight="0.79"
+                android:layout_gravity="left" />
         </LinearLayout>
 
         <LinearLayout
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
                 android:orientation="vertical"
-                android:layout_weight="11">
+                android:layout_weight="10.43">
```

Comment:
Leave this as 11, or maybe 10 if you feel the icons need more room.  The 11 here sets up the widths of the favicon view and the title/url/description view so they have a 1:11 width ratio.